### PR TITLE
Add io.js

### PR DESCRIPTION
**Pull request by zheller**
Add io.js for the community.

### CODE REVIEW

**Comment by ascandella** [comment]
shouldn't this be called `server.io`?

**Comment by zheller** [comment]
uh

**Comment by zheller** [comment]
no its io.js

**Comment by eklitzke** [comment]
i think it's supposed to be called `server.io`

File: server.io.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
```
Comment (eklitzke) [inline server.io.js]: this is wrong

**Pull request merged**